### DISCUSSION TITLE
Unstable scroll velocity on some views

### DISCUSSION THREAD

**Discussion by ElDonad** [community/community#127864] [Mobile] [answered]
It's a bit hard to describe the exact problem, but some scroll views (worst offender that I found the Explore tab) do not "launch" properly upon swipe : they loose almost all momentum at the end of the finger trajectory. This doesn't happen on other scroll views, like the feedback threads list.

Below, a link to a screen capture where this bug is showcased. I have tried to swipe at a consistent speed each time. 

https://github.com/community/community/assets/27915772/98e30525-a8a3-4694-8400-2415984a4efb

**Comment by elisealix22** [comment]
@ElDonad thank you so much for reaching out and providing that video! I opened an issue to track this internally and we will look into it. Please let us know if any other screens are slower to respond as well.

**Reply by alcere** [reply]
@ElDonad We recently shipped some improvements to the Explore view that should help. Could you please give it another try and let us know the results? Thank you!

**Reply by ElDonad** [reply, marked as answer]
Hi,

Thank you, it works as expected now !

Regards, 
Elie Donadio

**Comment by DRingOfficial** [comment]
Good
